include/odp/api/spec/traffic_mngr.h

```diff
@@ -189,6 +189,30 @@ extern "C" {
  * tree/hierarchy of nodes.
  */
 
+/** The tm_node_stats_type enumeration lists possible packet or octet
+ * statistics at a tm node.
+ */
+typedef enum odp_tm_node_stats_type_t {
+	/** Packets dropped by this node after scheduling/shaping at this node */
+	ODP_TM_NODE_STATS_PKTS_DROPPED,
+	/** Octets dropped after scheduling/shaping at this node */
+	ODP_TM_NODE_STATS_OCTETS_DROPPED,
+	/** Green packets that are sent through this tm node */
+	ODP_TM_NODE_STATS_GREEN_PKTS,
+	/** Green octets that are sent through this tm node */
+	ODP_TM_NODE_STATS_GREEN_OCTETS,
+	/** Yellow packets that are sent through this tm node */
+	ODP_TM_NODE_STATS_YELLOW_PKTS,
+	/** Yellow octets that are sent through this tm node */
+	ODP_TM_NODE_STATS_YELLOW_OCTETS,
+	/** Red packets that are sent through this tm node */
+	ODP_TM_NODE_STATS_RED_PKTS,
+	/** Red octets that are sent through this tm node */
+	ODP_TM_NODE_STATS_RED_OCTETS,
+	/** Node stats max */
+	ODP_TM_NODE_STATS_MAX,
+} odp_tm_node_stats_type_t;
```

Comment:
As discussed on tuesday, this way of defining the stats is different from other stats and the stats proposed in PR 1305. I actually like this way since it can make statistics handling in the application more streamlined, but we should not have two different ways of doing statistics within TM.

So, this PR and PR 1305 should converge to the same overall scheme of providing statistics and maybe also the actual counters should be the same in queue level (PR 1305) and in node level (this PR).

Reply:
Moved this patch to other pull request. Will provide new version there.